src/test/java/strategy/BalanceOperationTest.java

```diff
@@ -0,0 +1,35 @@
+package strategy;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import db.Storage;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class BalanceOperationTest {
+    private OperationHandler handler;
+
+    @BeforeEach
+    void setUp() {
+        handler = new BalanceOperation();
+    }
+
+    @AfterEach
+    void tearDown() {
+        Storage.getAll().clear();
```

Comment:
same for other places
```suggestion
        Storage.inventory.clear();
```